docs/python/generators/examples_helpers.py

```
@@ -57,35 +57,117 @@ def _find_unquoted_word(word: str, text: str) -> bool:
+    out = []
+    seen = set()
+    for n in (names_in or []):
+        if n and n not in seen:
+            seen.add(n)
+            out.append(n)
+    try:
+        from docs.python.generators import index as docs_index
+        for n in list(out):
+            try:
+                entry = docs_index.find_page(n)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)